python/tk_framework_desktopserver/settings.py

```diff
@@ -35,122 +34,97 @@ class Settings(object):
     _PORT_SETTING = "port"
     _CERTIFICATE_FOLDER_SETTING = "certificate_folder"
     _ENABLED = "enabled"
+    _HOST_ALIASES = "HostAliases"
 
-    def __init__(self, location, default_certificate_folder):
+    def __init__(self, default_certificate_folder):
```

Comment:
Since the browser integration API is not usable in standalone mode anymore, we can remove that bit of backwards compatibility.

Reply:
👍